Example/Podfile.lock

```diff
@@ -10,11 +10,11 @@ DEPENDENCIES:
 
 EXTERNAL SOURCES:
   Reusable:
-    :path: ../
+    :path: "../"
 
 SPEC CHECKSUMS:
   Reusable: 1775ede04719c488bb04e809d923cbb7214168b8
 
 PODFILE CHECKSUM: 396891106f5d114bc0d21fe52c4957aa880dd013
 
-COCOAPODS: 1.0.1
+COCOAPODS: 1.1.0.rc.2
```

Comment:
CocoaPodd 1.1.0 officially got released today, so that PR is the perfect occasion to use the non-RC version now :)